src/lib/simulation/toa.ts

```diff
@@ -320,6 +319,12 @@ const toaRequirements: {
 	{
 		name: 'Supplies',
 		doesMeet: ({ user, quantity }) => {
+			if (user.owns(new Bank().add('Super combat potion(4)', 1))) {
```

Comment:
The minimum amount of supplies needed should be static, this increments the amount with each toa command that is run

Reply:
Once I saw the main comment I figured this would be the issue will take a second look at this and make sure it's static instead of adding each time

Reply:
btw you can just use `user.owns('item name')` instead of `user.owns(new bank())`